src/Badges.sol

```diff
@@ -238,10 +250,17 @@ contract Badges is
     return isNotRevoked;
   }
 
-  function supportsInterface(bytes4 _interfaceId) public view virtual override returns (bool) {
+  function supportsInterface(bytes4 _interfaceId)
+    public
+    view
+    virtual
+    override(ERC165Upgradeable, IERC165Upgradeable)
+    returns (bool)
+  {
     return
       _interfaceId == type(IERC721Metadata).interfaceId ||
       _interfaceId == type(IERC4973).interfaceId ||
```

Comment:
Can we confirm that the interfaceId now matches what's in EIP spec, since we localized it? Do we have a test for this btw?

Reply:
We verified this off thread. Resolving this.